### PR TITLE
Issue 1

### PR DESCRIPTION
**Pull request by sandipdev2020**
Closes #X

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

**Comment by sandipdev2020** [comment]
[Ropes Issue 1](https://trello.com/c/CqxmZ6gy/1950-ropes-issue-1)

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
 
-    <%= javascript_include_tag 'application' %>
-    <%= javascript_pack_tag 'application' %>
+    <%#= javascript_include_tag 'application' %>
+    <%#= javascript_pack_tag 'application' %>
```
Comment (wanyue0714) [inline app/views/layouts/application.html.erb:26]: It's okay to remove them here, but later when you jump into React/Js issues you'll still need them. Just a reminder

File: test/controllers/application/application_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+class ApplicationControllerTest < ActionDispatch::IntegrationTest
+  def test_home
+    get root_path
+    assert_response :ok
+    #pp response.body
```
Comment (wanyue0714) [inline test/controllers/application/application_controller_test.rb:7]: Very minor, can we get rid of `pp response.body` here? Normally we don't leave comments in code, unless code itself is not that clear or cannot explain things well. But it's totally okay for ropes!

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,3 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def home
+    @name = 'Sandip'
```
Comment (wanyue0714) [inline app/controllers/application_controller.rb:5]: Not really related with current issue.
If you have `@name = 'Sandip'` here, then you could try using it on home page, sth like `<h1>Welcome to <%= @name %>'s ropes</h1>`, it should be `Welcome to Sandip's ropes`

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (sandipdev2020) [reply app/views/layouts/application.html.erb:26]: makes sense

File: test/controllers/application/application_controller_test.rb (hunk 2 above)
Comment (sandipdev2020) [reply test/controllers/application/application_controller_test.rb]: fixed :)

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (sandipdev2020) [reply app/controllers/application_controller.rb:5]: makes sense :)

**Pull request merged**